main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"runtime"
+
+	"github.com/fatih/color"
+	"github.com/ryanuber/columnize"
+
+	"github.com/giantswarm/gsctl/commands"
+)
+
+func init() {
+	columnizeConfig := columnize.DefaultConfig()
+	columnizeConfig.Glue = "   "
+
+	// disable color on windows, as it is super slow
+	if runtime.GOOS == "windows" {
+		color.NoColor = true
+	}
+}
+
+func main() {
+	// top level commands
+	commands.RootCommand.AddCommand(
```

Comment:
With cobra based applications, a pattern I'm more familiar with is only the root command being exported, and the subcommands adding themselves in their own `init` - it took me a while to find this.